bazaarvoice/emodb
# Pd 174110 update common jersey2 and BlobStore modules to use common-client with Javax 2.x

## Github Issue #

[Pd 174110](https://bazaarvoice.atlassian.net/browse/PD-174110)

## What Are We Doing Here?

1. Updating the base common-jersey2-client module of emodb and common-jersey2-client module of BlobStore to use newly added common-client module of javax 2.x

## How to Test and Verify

1. Check out this PR
2. Run Command `mvn dependency:tree` 

## Risk

### Level 

`Low`
